administrator/components/com_content/models/forms/article.xml

```diff
@@ -672,6 +672,15 @@
 			description="COM_CONTENT_FIELD_IMAGE_CAPTION_DESC"
 			size="20"/>
 	</fields>
+	<fields name="menulink" label="COM_CONTENT_MENU_LINK_OPTIONS">
+		<field
+				name="title"
+				type="text"
+				label="COM_CONTENT_ITEM_FIELD_TITLE_LABEL"
+				description="COM_CONTENT_ITEM_FIELD_TITLE_DESC"
+				class="input-xxlarge input-large-text"
+				size="40"/>
```

Comment:
Yes, phpcs will not complaint about it. But this is a common practice in joomla code to keep it in new line.